libtiny_logger/src/lib.rs

```diff
@@ -77,18 +88,35 @@ impl Logger {
 struct LoggerInner {
     /// Log file directory
     log_dir: PathBuf,
-
     /// Maps server names to their fds
     servers: HashMap<String, ServerLogs>,
-
     /// Callback used when reporting errors
     report_err: Box<dyn Fn(String)>,
 }
 
+#[derive(Debug)]
 struct ServerLogs {
```

Comment:
The inner map of `Logger` now works with `Option<File>` so configs can be stored even when logging is off for a particular server/chan